src/ContentFormHandlerBase.php

```diff
@@ -100,7 +127,7 @@ public function submitFormElements(array &$form, FormStateInterface $form_state)
       /** @var \Drupal\emr\Plugin\EntityMetaRelationPluginInterface $plugin */
       $plugin = $this->pluginManager->createInstance($id);
       if ($plugin instanceof EntityMetaRelationContentFormPluginInterface && $plugin->applies($entity)) {
-        $plugin->submit($form, $form_state);
+        $plugin->$callbackFunction($form, $form_state);
```

Comment:
Let's do a check here that the object (`$plugin`) has that callback function. I know, we control, but still.